delivery_cttexpress/models/delivery_carrier.py

```diff
@@ -127,10 +127,11 @@ def _onchange_cttexpress_shipping_type(self):
             )[self.cttexpress_shipping_type]
             raise UserError(
                 _(
-                    "This CTT Express service (%(service_name)s) isn't allowed for "
+                    "This CTT Express service (%(name)s) isn't allowed for "
```

Comment:
Can you avoid to change the translation and keep the same keyword arguments for not losing existing translations?